recipes/at-cascade/recipe.yaml

```
@@ -0,0 +1,73 @@
+schema_version: 1
+
+context:
+  # Version of package that this release corresponds to.
+  version: "2025.0.3"
+
+  # You may have to set python_min to get a local build to work but
+  # it is preferred to have python_min commented out when you commit this file.
+  # python_min: "3.10"
+
+package:
+  name: at-cascade
+  version: ${{ version }}
+
+source:
+  url: https://github.com/bradbell/at_cascade/archive/${{ version }}.tar.gz
+  sha256: f2c70fa39464f825134da09ab14fc671ffb5c74218809433ff962f92151aed6e
+
+build:
+  number: 0
+  noarch: python
+  script:
+    - ${{ PYTHON }} bin/check_py_test.py 
+    - ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - python =${{ python_min }}
+    - hatchling
+    - pip
+    - dismod-at >=2025.0.0
+    - numpy
+    - scipy
+  run:
+    - python =${{ python_min }}
```

```suggestion
    - python >=${{ python_min }}
```
For this we want to allow any version of python at or above the minimum, not just the minimum.